Source/wayland/WaylandServer.m

```diff
@@ -781,7 +781,7 @@ - (void)createSurfaceShell:(struct window *)window
       break;
     case NSPopUpMenuWindowLevel:
       NSDebugLog(@"[%d] NSPopUpMenuWindowLevel", win);
-      [self createPopup:win];
+      [self createPopup:get_window_with_id(wlconfig, win)];
```

Comment:
Why don't you use `window` here?

Reply:
This is fixed now.  Sorry I missed this.